swagger.json

```diff
@@ -180,7 +204,8 @@
                         "description": "The authorization header. There is a secret value that only application administrators know.",
                         "schema": {
                           "type": "string"
-                        }
+                        },
+                        "required": true
```

Comment:
I don't believe this header is actually required. You can also log in using a cookie.